template-helpers/homepage.js

```diff
@@ -5,6 +5,7 @@ const { LocaleUtils } = require("./../locale-utils");
 function makeLanding(args) {
   const locales = args.data.root.req.supportedLocales;
   const featuredBreach = args.data.root.featuredBreach;
+  const experimentBranchB = args.data.root.experimentBranchB;
```

Comment:
If we change the controllers to add an `experimentFlags` object to the template context, we'll need to update these references to `args.data.root.experimentFlags.experimentBranchB`, etc.